Widgets/AssetScrollAreaBackground.h

```diff
@@ -0,0 +1,60 @@
+#ifndef ASSETSCROLLAREABACKGROUND_H
+#define ASSETSCROLLAREABACKGROUND_H
+
+#include "AssetScrollArea.h"
+
+#include <QSet>
+#include <QWidget>
+
+#include <cmath>
+
+inline bool compareDouble(double a, double b) { return std::fabs(a - b) < std::numeric_limits<double>::epsilon(); }
+
+// RoundNum round to the nearest multiple
+template <typename T>
```

Comment:
This probably belongs in a utility header; here, we're just rounding points, so why not just accept a QPoint?

Reply:
I just stole this from a previous project. I dunno where else we would need this and we don't really have a utility header atm as qt does most our utility for us. I'll move it if and when I need it somewhere else. As for accepting qpoint I think it'd make for an ugly signature needing two rounder values accepted for each point. I think it's clearer what you're rounding x and y to this way